client/my-sites/hosting/sftp-card/index.js

```diff
@@ -143,27 +145,18 @@ const SftpCard = ( {
 				<div className="sftp-card__questions">
 					<Accordion title={ translate( 'What is SFTP?' ) }>
 						{ translate(
-							"{{p}}It's a way for you to access the files and folders on a website via a client program such as {{a}}Filezilla{{/a}} on your local computer.{{/p}}" +
-								'{{p}}SFTP stands for Secure File Transfer Protocol (or SSH File Transfer Protocol). It was designed as an extension of the SSH (Secure SHell) protocol. The “secure” part is because it is run over a secure channel, in this case SSH.{{/p}}' +
-								'{{p}}SFTP is not to be confused with File Transfer Protocol (FTP) which is similar but not secure. Thanks to the security provided by SFTP you can rest easy knowing that your files and your site are safe and sound.{{/p}}' +
-								'{{p}}A variety of custom plugins and themes may ask you to create specific folders or add files via SFTP however, generally speaking, SFTP is not required for your site to function.{{/p}}',
+							"It's a way for you to access the files and folders on a website via a client program such as {{a}}Filezilla{{/a}} on your local computer. " +
+								'For more information see {{supportLink}}SFTP and WordPress.com{{/supportLink}} ',
```

Comment:
The support page title is "SFTP on WordPress.com": https://en.support.wordpress.com/sftp/

```suggestion
								'For more information see {{supportLink}}SFTP on WordPress.com{{/supportLink}} ',
```

Reply:
Thanks! wow the GitHub suggestion feature is fancy